modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/SettingsModals/WhatsNewSettings.tsx

```diff
@@ -0,0 +1,284 @@
+			.active;
+
+		setPeriodButtons(periodOfTheDay);
+	};
+
+	const ScheduleUpdates = () => (
+		<div className="dashboard-whats-new-modal-container mt-3 pb-2 px-5">
+			<div className="font-weight-bolder"> Schedule</div>
+
+			<div className="font-weight-semi-bold mb-3 mt-3">
+				Generate list on
+			</div>
+
+			<div className="d-flex dashboard-whats-new-modal-buttons flex-row">
+				{buttons.map((button, index) => (
+					<div key={index}>
+						<ClayButton
+							className={classNames(
+								'mr-4 px-2 text-paragraph-sm text-uppercase dashboard-whats-new-modal-period-buttons',
+								{
+									'dashboard-whats-new-modal-active-buttons':
+										button.active,
+								}
+							)}
+							displayType="secondary"
+							onClick={() => {
+								selectCardDaysOfWeek(index);
+							}}
+						>
+							{button.name}
+						</ClayButton>
+					</div>
+				))}
+			</div>
+
+			<div className="font-weight-semi-bold mt-4">Data from</div>
+
+			<div className="d-flex dashboard-whats-new-modal-buttons dashboard-whats-new-modal-lines mt-3">
+				<ClaySelect
+					aria-label="Select Label"
+					className="dashboard-whats-new-modal-period-options mr-3"
+					id="mySelectId"
+				>
+					{dataFromOptions.map((item, index) => (
+						<ClaySelect.Option
+							key={index}
+							label={item.label}
+							value={item.value}
+						/>
+					))}
+				</ClaySelect>
+
+				<ClaySelect
+					aria-label="Select Label"
+					className="dashboard-whats-new-modal-options mr-3"
+					id="mySelectId"
+				>
+					{dataHoursOptions.map((item, index) => (
+						<ClaySelect.Option
+							key={index}
+							label={item.label}
+							value={item.value}
+						/>
+					))}
+				</ClaySelect>
+
+				{periodButtons.map((period, index) => {
+					return (
+						<div key={index}>
+							<ClayButton
+								className={classNames(
+									'mr-5 px-2 text-paragraph-sm text-uppercase dashboard-whats-new-modal-period-buttons',
+									{
+										'dashboard-whats-new-modal-active-buttons':
+											period.active,
+									}
+								)}
+								displayType="secondary"
+								onClick={() => {
+									selectPeriodCard(index);
+								}}
+							>
+								{period.name}
+							</ClayButton>
+						</div>
+					);
+				})}
+			</div>
+		</div>
+	);
+
+	return (
+		<div className="dashboard-whats-new-modal-container">
+			<div className="d-flex font-weight-bolder justify-content-between mt-5 pb-2 px-5">
+				Real time updates
+				<ClayToggle
+					onToggle={setRealUpdateToggled}
+					toggled={realUpdateToggled}
+				/>
+			</div>
+
+			<hr className="mb-5"></hr>
+
+			{realUpdateToggled === false && <ScheduleUpdates />}
+
+			<div className="font-weight-bolder mt-3 pb-2 px-5">Display</div>
+
+			<div className="font-weight-lighter mt-3 pb-5 px-5 text-neutral-8">
+				<div className="d-flex dashboard-whats-new-modal-lines dashboard-whats-new-modal-tittle justify-content-between mb-3">
```

Comment:
Please change the tittle name to title here and do the same everywhere it is used

```suggestion
				<div className="d-flex dashboard-whats-new-modal-lines dashboard-whats-new-modal-title justify-content-between mb-3">
```

Reply:
Done.